src/core-layers/grid-cell-layer/grid-cell-layer.js

```diff
@@ -177,6 +160,14 @@ export default class GridCellLayer extends Layer {
   }
 
   calculateInstancePositions64xyLow(attribute) {
+    const isFP64 = enable64bitSupport(this.props);
```

Comment:
Minor nit: my tastes would be to prefer a single comparison like this:
```
if (!enable64bitSupport(this.props)) {
  attribute.isGeneric = true;
  attribute.value = new Float32Array(2);
  return;
}

attribute.isGeneric = false;
...
```

Reply:
I've written it like this before, and I found it very easy to forget to reset `isGeneric` to `false` in 64bit mode. Debugging is hard when that happens. Maybe we need a better interface for generic attributes than a persistent flag which overrides the value format.